src/subgraph.js

```diff
@@ -134,11 +137,19 @@ Please update it to tell users more about your subgraph.`,
       .filter(dataSource => protocol.isValidKindName(dataSource.get('kind')))
       .reduce((errors, dataSource, dataSourceIndex) => {
         let path = ['dataSources', dataSourceIndex, 'mapping']
-
         let mapping = dataSource.get('mapping')
-
         const handlerTypes = protocolSubgraph.handlerTypes()
 
+        subgraphDebug(
+          'Validating dataSource "%s" handlers with %d handlers types defined for protocol',
+          dataSource.get('name'),
+          handlerTypes.size,
+        )
+
+        if (handlerTypes.size == 0) {
```

Comment:
@dotansimha  Substreams has no handlers, so needed to add that check to return earlier.